client/src/views/Report/DonutChart.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    PieChart,
+    Pie,
+    Cell,
+} from 'recharts';
+
+export default function DonutChart(props) {
+    const {
+        data,
+        colorScheme,
+    } = props;
+    return (
+        <PieChart
+            width={180}
+            height={180}
+        >
```

Comment:
Instead of using hardcoded values for width and height, it would be preferable to use ResponsiveContainer to make it responsive.